bin/cvejsontohtml.py

```diff
@@ -156,7 +156,10 @@ def getbasefor(fixedin):
                     continue
             allissues += "<li>Fixed in OpenSSL %s " %(fixedin)
             if (git != ""):
-                allissues += "<a href=\"%s\">(git commit)</a> " %(git)
+                if (fixedin.startswith("1.0.2") and fixedin[5]>='w'):  # 1.0.2w and above hack
+                    allissues += "<a href=\"/support/contracts.html?giturl=%s\">(premium support)</a> " %(git)
```

Comment:
👀 What is the `github=<commit-id>` parameter supposed to do?  I don't think we have anything that actually supports that...

Reply:
Aha ok, so just "documentation".  Got it